ui/raidboss/data/07-dt/eureka/occult_crescent_south_horn.ts

```diff
@@ -184,6 +246,580 @@ const triggerSet: TriggerSet<Data> = {
+            en: 'Tank Cleave on YOU, Avoid Line Stack',
+          },
+        };
+        data.deadStarsSliceTargets.push(matches.target);
+        if (data.deadStarsSliceTargets.length < 3)
+          return;
+
+        const target1 = data.deadStarsSliceTargets[0];
+        const target2 = data.deadStarsSliceTargets[1];
+        const target3 = data.deadStarsSliceTargets[2];
+        if (data.me === target1 || data.me === target2 || data.me === target3) {
+          if (!data.deadStarsIsSlice2)
+            return { alertText: output.tankCleaveOnYou!() };
+          return { alertText: output.tankCleaveOnYouLineStack!() };
+        }
+
+        // Do not call out with Firestrike 2
+        if (data.deadStarsIsSlice2)
+          return;
+
+        return {
+          infoText: output.tankCleavesOnPlayers!({
+            player1: data.party.member(target1),
+            player2: data.party.member(target2),
+            player3: data.party.member(target3),
+          }),
+        };
+      },
+      run: (data) => {
+        // Do not clear data for Firestrike 2 to use
+        if (data.deadStarsSliceTargets.length === 3 && !data.deadStarsIsSlice2) {
+          data.deadStarsSliceTargets = [];
+          data.deadStarsIsSlice2 = true;
+        }
+      },
+    },
+    {
+      id: 'Occult Crescent Dead Stars Primordial Chaos',
+      // Each boss targets ground, avoid getting hit by more than one aoe
+      // A5D9 Primordial Chaos castbar
+      // A5DC Primordial Chaos damage cast for each alliance
+      type: 'StartsUsing',
+      netRegex: { source: 'Phobos', id: 'A5D9', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Occult Crescent Dead Stars Nova/Ice Ooze Initial',
+      // This won't work until FFXIVACT Plugin captures StatusEffectListForay3
+      // Applied with Primordial Chaos
+      // Comes in stacks of 1, 2, or 3
+      // 1159 Nova Ooze (Red)
+      // 115A Ice Ooze (Blue)
+      // Players need to get hit by opposite color Ooze to decrease count
+      // Hits by same color Oooze will increase count
+      // Four opportunities to increase/decrease stack, meaning those with lower counts can afford mistakes
+      // Any stacks remaining before Noxious Nova (A5E5) result in lethal damage
+      type: 'GainsEffect',
+      netRegex: { effectId: ['1159', '115A'], capture: true },
+      condition: (data, matches) => {
+        if (data.me === matches.target)
+          return true;
+        return false;
+      },
+      suppressSeconds: 60, // Ignore during mechanic
+      infoText: (_data, matches, output) => {
+        const num = matches.count;
+        if (matches.effectId === '1159')
+          return output.blue!({ num: num });
+        return output.red!({ num: num });
+      },
+      outputStrings: {
+        red: {
+          en: 'Get hit by red ${num}x',
+        },
+        blue: {
+          en: 'Get hit by blue ${num}x',
+        },
+      },
+    },
```

Comment:
I think we should keep triggers that rely on the missing `StatusEffectListForay3` packet completely commented out for now.  If we merge these in and they don't function, users will still see the triggers in the config UI and wonder why they aren't working.  I think it's fine to have the code here as a comment for future reference though.